uninstall.sh

```diff
@@ -404,21 +413,10 @@ uninstall() {
     git config --global --unset githooks.previous.searchdir
     git config --global --unset githooks.disable
     git config --global --unset githooks.installDir
+    git config --global --unset githooks.pathForUseCoreHooksPath
+    git config --global --unset githooks.useCoreHooksPath
     git config --global --unset alias.hooks
 
-    if using_hooks_path; then
```

Comment:
No ifs, rather prefere to uninstall all stuff always (its an uninstall, try not to leave stuff behind -> which is not guaranteed too, but that needs fixes as well, maybe somewhen else), moved the code here to the top, and the unset of configs together with the others